.npmrc

```diff
@@ -0,0 +1,2 @@
+save-exact=true
+legacy-peer-deps=true
```

Comment:
hmm, I wonder why this is required in a new project 🤔 But let's not spend time on this as it's not that important to our goal